concepts/80 Troubleshooting/05 DataGrid and TreeList Issues/10 Gray Boxes Appear After Data Loading/25 Eliminate Side Effects.md

```diff
@@ -0,0 +1,103 @@
+Follow the steps below to eliminate possible side effects:
+
+- Disable the [stateStoring](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/stateStoring/) property. Check if the issue still persists. If the issue disappears, it is possible that the saved state is corrupted. Clear the saved state, enable **stateStoring**, and check again.
+
+- FireFox and Safari browsers raise native scrolling events asynchronously. This behavior forces asynchronous row rendering in native scrolling mode even when [renderAsync](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#renderAsync) is disabled. To avoid this side effect, disable the scrolling.[useNative](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#useNative) property. Call the [defaultOptions(rule)](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#defaultOptionsrule) method to eliminate the issue for all DataGrid components in the application.
```

Comment:
```suggestion
- FireFox and Safari browsers raise native scrolling events asynchronously. This behavior forces asynchronous row rendering in native scrolling mode even when [renderAsync](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#renderAsync) is disabled. To avoid this side effect, disable the scrolling.[useNative](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#useNative) property. Call the [defaultOptions(rule)](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#defaultOptionsrule) method to resolve the issue for all DataGrid components in the application.
```